app/views/notifiers/course/consolidated_opening_reminder_notifier/opening_reminder/course_notifications/email.html.slim

```diff
@@ -0,0 +1,13 @@
+- course = @object
+- items_hash = Course::LessonPlan::Item.upcoming_items_from_course_by_type(course)
+/ Lesson plan item start at times could have been changed between the time the mailer job was
+/ enqueued and the time this view partial is called.
+/ Raise an exception if there are no longer any items so the consolidated email doesn't get sent.
+- raise 'No opening lesson plan items' if items_hash.empty?
+
+- message.subject = t('.subject', course: format_inline_text(course.title))
+p = t('.message')
+
+- items_hash.sort.each do |actable_type, items|
```

Comment:
Just to check, is it possible for `items` to be an empty array for some weird reason? Was thinking if we should skip if `items` is empty in case the header is rendered (without any item).

Reply:
Shouldn't be. The hash keys (actable types) are derived from the list of items, so if there's no item of that type, there wouldn't be a hash key for that type at all.